sroka/api/mysql/README.md

```diff
@@ -0,0 +1,46 @@
+# MySQL connector
+
+## Configuration
+
+Here are the configuration variables that the `mysql` connector supports:
+
+* `host`: The host address on which the MySQL server you want to access runs. Can be an IP or a domain (e.g.: `127.0.0.1`, `localhost`, `test.com/mysql`, etc.). Mutually exclusive with the `unix_socket` option.
```

Comment:
They are mutually exclusive so they shouldn't ever be specified together. In the MySQL connector implementation, it seems like [Unix Sockets have the priority though](https://github.com/mysql/mysql-connector-python/blob/b034f25ec8037f5d60015bf2ed4ee278ec12fd17/lib/mysql/connector/connection.py#L264-L269).

However, I think documenting which is prioritized would encourage misusing the connector and specifying both when only one is needed, what do you think? Should it be mentioned anyway? Should we make it clearer that they are mutually exclusive?

Reply:
So maybe we should do a check and if both are non-empty print a warning? And write which one was used in this case?

Reply:
Currently what I did was to actually error during the configuration validation if both are set, which I'd argue is less error-prone since it ensures that people never use an ambiguous configuration, what do you think?

Reply:
That is fine with me 👍

Reply:
I guess this is handles now by the code, so additional clarification is not needed.